<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastFMAuthDialog</class>
 <widget class="QDialog" name="LastFMAuthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scrobbling</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Username:</string>
       </property>
       <property name="buddy">
        <cstring>username</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="username"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishLoginButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Finish login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="logoutButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>LastFMAuthDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
